domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/induction/service/CiagKpiService.kt

```diff
@@ -1,37 +1,15 @@
 package uk.gov.justice.digital.hmpps.domain.learningandworkprogress.induction.service
 
-import mu.KotlinLogging
-import uk.gov.justice.digital.hmpps.domain.learningandworkprogress.induction.InductionScheduleCalculationRule
 import java.time.Instant
 import java.time.LocalDate
 
 /**
  * Abstract superclass for CIAG KPI service methods
  */
 
-private val log = KotlinLogging.logger {}
 abstract class CiagKpiService {
 
   abstract fun processPrisonerAdmission(prisonNumber: String, prisonAdmittedTo: String, eventDate: Instant)
-  fun processPrisonerTransfer(prisonNumber: String, prisonTransferredTo: String) {
-    // TODO - implement
-  }
-
-  fun determineInductionScheduleCalculationRule(prisonNumber: String): InductionScheduleCalculationRule {
```

Comment:
This was never called by anything, and now would live in the `InductionScheduleDateCalculationService` class.
At the moment I've just removed it as nothing was calling it and most of it is commented out.
As and when we need it (or something similar) I think we'll end up doing it in `InductionScheduleDateCalculationService`